source/drhtaxes/corporate.py

```diff
@@ -0,0 +1,11 @@
+class COR(object):
+
```

Comment:
It would be nice to have some documentation on what this class does. Using the """ ... """ type documentation we talked about earlier.